lego/apps/banners/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+from lego.apps.banners.permissions import BannersPermissionHandler
+from lego.utils.models import BasisModel
+
+
+class Banners(BasisModel):
+    header = models.CharField()
+    subheader = models.CharField()
+    link = models.CharField()
+    current_public = models.BooleanField(default=False, null=False, blank=False)
+    current_private = models.BooleanField(default=False, null=False, blank=False)
+
+    class Meta:
+        permission_handler = BannersPermissionHandler()
+        indexes = [
+            models.Index(fields=["current_public"]),
+            models.Index(fields=["current_private"]),
+        ]
```

Comment:
Just a thought, but I think it might be possible to add `meta.constraints` to ensure only ever one instance ever can have `current_public` set to true. It is strictly not needed if it is always handled correct, but could maybe be nice to enforce it on a database-level